Tofu/AccountsViewController.swift

```diff
@@ -76,12 +80,19 @@ class AccountsViewController: UITableViewController {
 
     private func makeAddAccountAlertController() -> UIAlertController {
         let title = "Add Account"
-        let message = "Add an account by scanning a QR code or enter a secret manually."
+        let message = "Add an account by scanning a QR code, importing a QR image, or entering a secret manually."
         let alertController = UIAlertController(title: title, message: message, preferredStyle: .actionSheet)
 
         let scanQRCode = UIAlertAction(title: "Scan QR Code", style: .default) { [unowned self] _ in
             self.performSegue(withIdentifier: "ScanSegue", sender: self)
         }
+        
+        let importQRCode = UIAlertAction(title: "Import QR Image", style: .default) { [unowned self] _ in
+            self.qrImporter.allowsEditing = false
+            self.qrImporter.sourceType = .photoLibrary
+                
+            self.present(self.qrImporter, animated: true, completion: nil)
```

Comment:
Did you consider calling [`isSourceTypeAvailable(_:)`](https://developer.apple.com/documentation/uikit/uiimagepickercontroller/1619144-issourcetypeavailable) to check if we can pick from the photo library before presenting this controller as suggested by the [docs on UIImagePickerController](https://developer.apple.com/documentation/uikit/uiimagepickercontroller)? I'm not sure what happens if we present it anyway and the docs don't seem to mention this.

Reply:
Ah, no I hadn't. I hadn't actually considered instances where it wouldn't be available -- that's good to know for future reference though. Just added it in.